client/control/components/common/DeprecationAlert.js

```diff
@@ -0,0 +1,86 @@
+import { Alert } from 'antd';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+
+const ENV_PROD = 'ENV_PROD';
+const ENV_STAGE = 'ENV_STAGE';
+const ENV_DEV = 'ENV_DEV';
+
+const DC_URLS = {
+  [ENV_PROD]: 'https://delivery-console.prod.mozaws.net/',
+  [ENV_STAGE]: 'https://delivery-console.stage.mozaws.net/',
+  [ENV_DEV]: 'https://delivery-console.dev.mozaws.net/',
+};
+
+/**
+ * Simple component which tells the user whether they are viewing a dev or staging
+ * environment. On production, nothing is shown.
+ */
+@connect(() => ({
+  currentUrl: window.location.href,
```

Comment:
Isn't there a selector that does path information? Or is that only in DC?

Also, if this is the only thing being passed it, I don't think it needs to be a connected component.